docs/cpp/lldb-mi.md

```diff
@@ -0,0 +1,77 @@
+---
+Order: 15
+Area: cpp
+TOCTitle: Setup Help for LLDB-MI
+ContentId: 1e84d196-397f-4dbb-9746-06f15766d83e
+PageTitle: How to setup debugging on macOS with LLDB-MI
```

Comment:
I think `set up` is two words in this context, which is different from the TOCTitle